plugins/modules/cargo.py

```diff
@@ -236,6 +253,7 @@ def main():
         version=dict(default=None, type="str"),
         locked=dict(default=False, type="bool"),
         directory=dict(default=None, type="path"),
+        features=dict(default=[], required=False, type="list", elements="str"),
```

Comment:
I think the default `[]` for list is better than `None`. From what I've seen without specifying it in the specs it defaults to `None`.

Reply:
Indeed, without a default value that parameter will still exist but it will hold `None` as value.

As to using `[]` instead of `None`, it does make sense to me also, as it simplifies the code later on.